components/record/MetadataField.vue

```diff
@@ -22,6 +24,15 @@
       value: {
         type: [String, Object],
         default: ''
+      },
+      context: {
```

Comment:
Should we give this a [validator](https://vuejs.org/v2/guide/components-props.html#Prop-Validation) to ensure it's a known context?

Reply:
Could do, but then I'd be tempted to get the available contexts from the labelling plugin via say `fieldLabelContexts` and while that would work, I think it unnecessarily ties the context to be specific to labelling. Perhaps we want contexts for the metadata fields that apply [case filters](https://vuejs.org/v2/guide/filters.html) in future.
The labelling will fall back to the default context as is, so if an unknown labeling context would be applied now I don't see it as worth raising console errors, just let it show the default labels from the record or fall back to the key.